src/mlpack/core.hpp

```diff
@@ -71,45 +71,22 @@
 #include <mlpack/core/util/log.hpp>
 #include <mlpack/core/util/io.hpp>
 #include <mlpack/core/util/deprecated.hpp>
-#include <mlpack/core/data/load.hpp>
-#include <mlpack/core/data/save.hpp>
-#include <mlpack/core/data/normalize_labels.hpp>
-#include <mlpack/core/math/clamp.hpp>
-#include <mlpack/core/math/random.hpp>
-#include <mlpack/core/math/random_basis.hpp>
-#include <mlpack/core/math/lin_alg.hpp>
-#include <mlpack/core/math/range.hpp>
-#include <mlpack/core/math/round.hpp>
-#include <mlpack/core/math/shuffle_data.hpp>
-#include <mlpack/core/math/ccov.hpp>
-#include <mlpack/core/math/make_alias.hpp>
-#include <mlpack/core/math/quantile.hpp>
-#include <mlpack/core/dists/discrete_distribution.hpp>
-#include <mlpack/core/dists/gaussian_distribution.hpp>
-#include <mlpack/core/dists/laplace_distribution.hpp>
-#include <mlpack/core/dists/gamma_distribution.hpp>
-#include <mlpack/core/dists/diagonal_gaussian_distribution.hpp>
-#include <mlpack/core/data/confusion_matrix.hpp>
-#include <mlpack/core/data/one_hot_encoding.hpp>
+#include <mlpack/core/data/data.hpp>
+#include <mlpack/core/math/math.hpp>
 
 // mlpack::backtrace only for linux
 #ifdef HAS_BFD_DL
```

Comment:
@rcurtin suggest to use `MLPACK_HAS_BFD_DL` instead of `HAS_BFD_DL`. This is to avoid potential clashes with headers from other software that users may include.

Reply:
Ah, I guess I missed this in a previous PR.  Done in c973ce7.